opinions/united_states/state/tex.py

```diff
@@ -24,16 +31,18 @@
 
 
 class Site(OpinionSite):
+    back_scrape_iterable = [i.date() for i in rrule(
+        YEARLY,
+        dtstart=date(1981, 1, 1),
+        until=date(2015, 8, 30),
```

Comment:
I'm unclear...since we're doing `YEARLY`, will this include 2015 or not? Can we just set it to 2015-01-01 and achieve the same thing more clearly?